projects/RabbitMQ.Client/client/api/PublicationAddress.cs

```diff
@@ -116,6 +116,22 @@ public static PublicationAddress Parse(string uriLikeString)
             return null;
         }
 
+        public static PublicationAddress TryParse(string uriLikeString) {
+            // Callers such as IBasicProperties.ReplyToAddress
+            // expect null result for invalid input.
+            // The regex.Match() throws on null arguments so we perform explicit check here
+            if (uriLikeString == null)
+            {
+                return null;
+            } else {
```

Comment:
I'm not aware of the coding guidelines for the repo but rest of the code uses places `{` braces on a new line.